pkgs/development/python-modules/darkdetect/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+
+, glib
+, setuptools
+}:
+
+buildPythonPackage rec {
+  pname = "darkdetect";
+  version = "0.8.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "albertosottile";
+    repo = "darkdetect";
+    rev = "v${version}";
+    hash = "sha256-OOINgrgjSLr3L07E9zf1+mlTPr+7ZlgN3CfkWE8+LoE=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+  ];
+
+  pythonImportsCheck = [
+    "darkdetect"
+  ];
+
+  postPatch = ''
+    substituteInPlace darkdetect/_linux_detect.py \
+      --replace "gsettings" "${glib.bin}/bin/gsettings"
```

Comment:
```suggestion
      --replace '"gsettings"' '"${glib.bin}/bin/gsettings"'
```

Reply:
They use singular quotes. If that suggestion is because you worry about replacing something in the code that shouldn't be replaced, I'll do the opposite wrapping, then.

Reply:
great